File: packages/tasks-gen/snippets-fixtures/text-to-speech/js/huggingface.js/0.fal-ai.js
import { InferenceClient } from "@huggingface/inference";

const client = new InferenceClient(process.env.HF_TOKEN);

const audio = await client.textToSpeech({
    provider: "fal-ai",
    model: "nari-labs/Dia-1.6B",
	inputs: "The answer to the universe is 42",
});
// Use the generated audio (it's a Blob)

File: packages/tasks-gen/snippets-fixtures/text-to-speech/python/huggingface_hub/0.fal-ai.py
import os
from huggingface_hub import InferenceClient

client = InferenceClient(
    provider="fal-ai",
    api_key=os.environ["HF_TOKEN"],
)

# audio is returned as bytes
audio = client.text_to_speech(
    "The answer to the universe is 42",
    model="nari-labs/Dia-1.6B",
)